lib/ts/recipe/thirdpartypasswordless/recipe.ts

```diff
@@ -113,72 +105,6 @@ export default class Recipe extends RecipeModule {
                 ? new SmsDeliveryIngredient(this.config.getSmsDeliveryConfig())
                 : ingredients.smsDelivery;
 
-        this.emailVerificationRecipe =
```

Comment:
This file still has `getEmailForUserIdForEmailVerification`  - is this function needed?